ios_ui/MovieRec/MoviesToRate.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  MoviesToRate.swift
+//  MovieRec
+//
+//  Created by Yujin Cho on 8/18/17.
+//  Copyright © 2017 Yujin Cho. All rights reserved.
+//
+
+import os.log
+import UIKit
+
+class MoviesToRate {
+    
+    var userId: String?
+    var movies = [Movie]()
+    
+    init() {
+        setUserID()
+        loadMovies()
+        //print(movies.map {(movie:Movie) -> String in movie.title})
+        print("Movie Count: \(movies.count)")
+    }
+    
+    func count() -> Int {
```

Comment:
I prefer to use a computed property for these types of things that aren't really methods
var count: Int { return ... }